tools/lint.py

```diff
@@ -116,9 +116,9 @@ def main():
     else:
         files = args.files
 
-    cython_exts = ['.pyx']  # Add '.pxd', '.pxi' once cython-lint supports it
+    cython_exts = ('.pyx', '.pxd', '.pxi')
     cython_files = [f for f in files if any(f.endswith(ext) for ext in cython_exts)]
-    other_files = [f for f in files if not f.endswith('.pyx')]
+    other_files = [f for f in files if not any(f.endswith(ext) for ext in cython_exts)]
```

Comment:
Looks good, thanks! Another alternative would be to use set logic.

Reply:
Just pushed -- did you have something like that in mind?

Reply:
Thanks, yes, that's it!